app/serializers/user_serializer.rb

```diff
@@ -274,6 +276,14 @@ def stats
     UserAction.stats(object.id, scope)
   end
 
+  def ignored
+    IgnoredUser.find_by(user_id: scope.user&.id, ignored_user_id: object.id).present?
```

Comment:
This is slightly faster and requires less memory since it doesn't load the whole object in memory ;)

```suggestion
    IgnoredUser.where(user_id: scope.user&.id, ignored_user_id: object.id).exists?
```